brands/itel/README.md

```diff
@@ -1,12 +1,17 @@
 # itel 
 
-* Verdict **ℹ️ "Safe for now" :trollface:**
+- Verdict: **⚠️ Proceed with caution!**
+
 * [**🔓️ Unlock Guide (MediaTek)**](/misc/generic-unlock.md)
 * [**🔓️ Unlock Guide (Unisoc)**][Unisoc Unlock]
 
+Itel requires you to have a Itel ID for two weeks before the OEM Unlocking option becomes available in the settings. Afterwards, you can follow the [standard unlock guide](/misc/generic-unlock.md) for devices with a MediaTek SoC, or the [Unisoc unlock guide][Unisoc Unlock] for models with a Unisoc SoC.
```

Comment:
I can't find a source for this online.

Reply:
I bought their phone a week ago :-(

But I should google before buying. For instance,
https://xdaforums.com/t/itel-id-preventing-me-from-turning-on-oem-unlocking-as-my-account-is-not-over-2-weeks-old.4675087/
(see the date - earlier Itel didn't do such things)

By the way, similar to your list:
https://en.wikipedia.org/wiki/Bootloader_unlocking
"Itel ... requires Itel ID account on the phone, registered in two weeks or more, to activate OEM unlocking"

Reply:
One of those should be linked on the page.

Reply:
Ok, I added link to Wiki to the page